readthedocs/api/v3/serializers.py

```diff
@@ -656,6 +650,11 @@ class Meta:
             ),
         }
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if settings.RTD_ALLOW_ORGANIZATIONS:
+            self.fields.pop("users", None)
```

Comment:
This needs a comment. I assume this is the same logic as before, but inverted? So we remove users when the site allows orgs?

Reply:
yeah, this is to make it easier to test, otherwise the attributes defined at the class level wouldn't change (they were defined at import time, now they are defined at instance time)